ibm/mas_devops/roles/suite_manage_load_dbc_scripts/tasks/main.yaml

```diff
@@ -49,6 +49,7 @@
       debug:
         msg:
           - "Instance ID ............... {{ mas_instance_id }}"
+          - "App ID ............... {{ mas_app_id }}"
```

Comment:
Would you mind fixing the standard dots in the debug just to keep it consistent with other lines?

Reply:
Added